guides/auth-torii-with-github.md

```diff
@@ -0,0 +1,605 @@
+[Back to Main README](../README.md)
+
+# Full GitHub Authorization with the `torii-provider`
+
+This guide walks through the process of authenticating and authorizing an Ember app with
+and for use against GitHub using `ember-simple-auth` and `torii` including a registered
+GitHub application, an unauthenticated login route, an authenticated content route, a
+token exchange service, and use of an authorizer to inject the token into
+requests. We'll use the [ember-data-github](https://github.com/elwayman02/ember-data-github)
+addon so we don't need to create our own GitHub models, serializers, and adapters.
```

Comment:
I think it would be good to briefly explain that the general concepts are the same for any other OAuth 2.0 provider (Facebook, Google etc.). Otherwise people who try to use one of those providers might feel that this wouldn't help them while it actually would. Maybe we could even rename the whole guide to _"Implementing the OAuth 2.0 Authorization Code Grant Type with Ember Simple Auth and torii"_ or so and simply explain in the guide that github is only used as an example.